docker/deployment/entrypoint.sh

```diff
@@ -6,15 +6,56 @@ if [ "$#" -gt 0 ]; then
     shift
 fi
 
+_octane() {
+  # exit on config update fail
+  set -e
+
+  /app/artisan config:cache
+  /app/artisan route:cache
+
+  /app/docker/deployment/entrypoint.sh config_reloader &
+  /app/artisan octane:start --host=0.0.0.0 &
+  OCTANE_PID=$!
+
+  wait $OCTANE_PID
+
+  echo "octane exit $?"
+
+  # kill child processes (config reloader)
+  pkill -TERM -P $$
+}
+
+_config_reloader() {
+  while true; do
+    echo "Awaiting config update on .env..."
+    inotifywait -e modify .env
+
+    echo "Reloading config..."
+    /app/artisan config:cache
+    /app/artisan route:cache
+    /app/artisan octane:reload
+  done
+}
+
 _php() {
     /app/artisan config:cache
     /app/artisan route:cache
     exec php-fpm8.0 -y /app/docker/deployment/php-fpm.conf
 }
 
+_schedule() {
```

Comment:
is there no existing schedule function on kubernetes or related management tool? This will have the box doing nothing but `sleep` most of the time.

Reply:
There is, I am not actually using this in Kubernetes deploys. This exists because I went for a secondary docker-compose that was more production-like at the beginning, as I did not want to dive in blindly.

I will remove the loop and just call schedule:run.